src/SerialPortComms.cs

```diff
@@ -1,54 +1,50 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO.Ports;
-using System.Linq;
-using System.Text;
 using System.Threading;
-using System.Threading.Tasks;
 
 namespace COMSpike
 {
     public class SerialPortComms
     {
-        private SerialPort port;
+        private readonly SerialPort _port;
 
         public SerialPortComms(int portNumber)
         {
-            port = new SerialPort($"COM{portNumber}", 9600, Parity.None, 8, StopBits.One);
+            _port = new SerialPort($"COM{portNumber}", 9600, Parity.None, 8, StopBits.One);
 
-            port.DataReceived += Port_DataReceived;
-            port.ErrorReceived += Port_ErrorReceived;
+            _port.DataReceived += Port_DataReceived;
+            _port.ErrorReceived += Port_ErrorReceived;
         }
 
         public void ResetWhoop()
         {
-            port.Open();
+            _port.Open();
 
-            Console.WriteLine($"IsOpen: {port.IsOpen}");
+            Console.WriteLine($"IsOpen: {_port.IsOpen}");
 
-            Console.WriteLine("Writing 'Reboot' to port...");
-            port.Write("Reboot");
+            Console.WriteLine("Writing 'Reboot' to _port...");
+            _port.Write("Reboot");
 
             Thread.Sleep(1000);
 
             Close();
         }
 
-        internal void Close()
+        private void Close()
         {
-            Console.WriteLine("Closing port...");
-            port.Close();
+            Console.WriteLine("Closing _port...");
+            _port.Close();
         }
 
-        private void Port_ErrorReceived(object sender, SerialErrorReceivedEventArgs e)
+        private static void Port_ErrorReceived(object sender, SerialErrorReceivedEventArgs e)
         {
             Console.WriteLine($"Error Received: {e.EventType}");
         }
 
         private void Port_DataReceived(object sender, SerialDataReceivedEventArgs e)
         {
-            // Show all the incoming data in the port's buffer
-            Console.WriteLine($"Data Received: {port.ReadExisting()}");
+            // Show all the incoming data in the _port's buffer
```

Comment:
Search and replace gone wrong

Reply:
whoops, I trusted VS 2022 to do the job. 😆